src/node_api_helpers.h

```diff
@@ -14,6 +14,7 @@
 #include "node_asyncapi.h"
 #include <limits.h>
 #include <string.h>
+#include <assert.h>
```

Comment:
It's used below in existing code and, when you include this header, your code fails to build if you don't make this modification.